packages/babel-plugin-transform-parameters/test/fixtures/regression/6057-expanded/output.js

```diff
@@ -9,8 +11,6 @@ var _args = _interopRequireDefault(require("utils/url/args"));
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
-function _typeof(obj) { "@babel/helpers - typeof"; if (typeof Symbol === "function" && typeof Symbol.iterator === "symbol") { _typeof = function _typeof(obj) { return typeof obj; }; } else { _typeof = function _typeof(obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; }; } return _typeof(obj); }
```

Comment:
Ironically, this test was about another `context` caching bug (https://github.com/babel/babel/issues/6057), but this change has nothing to do with that bug.